examples/postgresql/README.md

```diff
@@ -13,7 +13,7 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
 - Kubernetes 1.10+
```

Comment:
```suggestion
- Kubernetes 1.20+
```
Let's bump this up since we don't really support older versions

Reply:
fixed